src/services/summary/summary-model.js

```diff
@@ -0,0 +1,36 @@
+const mongoose = require('mongoose');
+
+const Schema = mongoose.Schema;
+
+const byDepartmentSchema = new Schema({
+  department: { type: Schema.Types.ObjectId, ref: 'Department', required: true },
+  assigned: { type: Number },
+  processing: { type: Number },
+  resolved: { type: Number },
+});
+
+const summarySchema = new Schema({
```

Comment:
My plan is to use activity log to recalculate the number of each day. It won't be real-time for now. You would have to trigger it through `triggter` routing that I am gonna make. WDYT?